cmake/BuildParameters.cmake

```diff
@@ -349,6 +351,22 @@ else()
     set(LTO_FLAGS "")
 endif()
 
+if (USE_PGO_GENERATE)
+    message(WARNING "PGO has not been thoroughly tested. A bug is not impossible.")
+    if(USE_PGO_OPTIMIZE)
+        message(WARNING "PGO_OPTIMIZE and PGO_GENERATE should not be used together! Do so at your own risk!")
+        set(PGO_FLAGS "-fprofile-dir=${CMAKE_SOURCE_DIR}/profile -fprofile-generate -fprofile-use")
```

Comment:
I'm not sure the variable is the good one. What is the value of CMAKE_SOURCE_DIR if you build in the build_dbg directory?
Add a side note, it would be nice to extend the build.sh script too.